tests/system/test_dbapi.py

```diff
@@ -57,6 +58,7 @@ def raw_database(shared_instance, database_operation_timeout, not_postgres):
         database_id,
         ddl_statements=DDL_STATEMENTS,
         pool=pool,
+        database_dialect=DatabaseDialect.GOOGLE_STANDARD_SQL,
```

Comment:
Why are we setting this here? Shouldn't the database just read it itself from the server?

Reply:
Yes, removed. Thanks